packages/components/ng-ui-router-layout/src/oui-modal/template.html

```diff
@@ -0,0 +1,36 @@
+<div data-modal-render="{{$isRendered}}" tabindex="-1" role="dialog" class="modal"
+     data-uib-modal-animation-class="fade"
+     data-modal-in-class="in"
+     data-ng-style="{'z-index': 1050 + index*10, display: 'block'}">
+    <div class="modal-dialog {{size ? 'modal-' + size : ''}}">
+        <div class="modal-content">
+            <oui-modal
```

Comment:
Regarding the template of the [`<oui-modal>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/modal) component, it seems that you have to wrap it inside a `<form>` element.

See: https://github.com/ovh-ux/ovh-ui-angular/blob/d7f863572ff0a6aa172955942d96df24f60bc138/packages/oui-modal/src/modal.html#L36

Am I wrong?

Reply:
I works without but yes it could.
So the ngSubmit on the form is the primaryAction callback then?